backend/icpy/agent/tools/atlascloud/itv_tool.py

```
@@ -0,0 +1,610 @@
+            
+            # If path is already absolute, use it as-is, otherwise join with workspace root
+            if os.path.isabs(clean_image):
+                full_path = Path(clean_image)
+            else:
+                full_path = Path(workspace_root) / clean_image
+        else:
+            # Remote context
+            full_path = Path(clean_image)
+        
+        if not full_path.exists():
+            raise ValueError(f"Image file not found: {image} (resolved to {full_path})")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)